wdl/tasks/TertiaryAnalysis/FunctionalAnnotation.wdl

```diff
@@ -0,0 +1,61 @@
+version 1.0
```

Comment:
minor: I actually begin to think that we should have a `Annotation` folder now under `wdl/tasks/`

Reply:
Yeah - we can refactor this later.  Probably worth a discussion about how to categorize some of these WDLs.